src/PPU.cpp

```diff
@@ -347,48 +353,147 @@ void PPU::updateObjects() {
 	}
 }
 
+void PPU::updateBackgroundScanline(uint32_t* buffer, int* startingIndex) {
+	uint8_t scx = mmu->directRead(0xFF43);
+	uint8_t scy = mmu->directRead(0xFF42);
+	
+	// Find y coordinate of the tile in the background map
+	uint8_t y = (ly + scy) % 256;
+	uint16_t tileY = y / 8;
+	uint16_t tileYPos = tileY * 32;
+
+	// Starting index is the offset within the first tile to start reading
+	*startingIndex = scx & 0x07;
```

Comment:
I was seeing some weird perf results when putting this right after the scx and scy reads. Like x64 release, ~730 fps with it above and ~780 fps above. Not sure if that's real or if it's a result of something else.